esmvaltool/recipes/recipe_carvalhais2014nat.yml

```diff
@@ -0,0 +1,135 @@
+# ESMValTool
+# recipe_carvalhais2014nat.yml
+---
+documentation:
+  description: |
+    Recipe to evaluate the ecosystem turnover time of carbon based on
+    Carvalhais et al., 2014, Nature.
+
+  authors:
+    - koirala_sujan
+
+  maintainer:
+    - koirala_sujan
+
+  references:
+    - carvalhais14nature
+
+  projects:
+    - crescendo
+
+
+datasets:
+
+  - {dataset: HadGEM2-ES, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: MIROC-ESM, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: IPSL-CM5A-MR, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: NorESM1-M, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: MPI-ESM-LR, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: CCSM4, project: CMIP5, ensemble: r1i1p1}
+  - {dataset: GFDL-ESM2G, project: CMIP5, ensemble: r1i1p1}
+
+
+obs_details: &obs_details
+  obs_info:
+    #
+    # size of the big box in the latitudinal direction.
+    # Recommended value: 5
+    obs_data_subdir: 'data_ESMValTool_Carvalhais2014'
+    source_label: 'Carvalhais2014'
+    variant_label: 'BE'
+    grid_label: 'gn'
+    grid_extent: '0.5x0.5'
+    # grid_label: 'gr'
+    # grid_extent: '2.5x1.875'
+    # 'gn': 0.5x0.5, native observation
+    # 'gr1': 2.812x2.813, bcc-csm1-1
+    # 'gr2': 1.25x0.937, CCSM4
+    # 'gr3': 2.812x2.813, CanESM2
+    # 'gr4': 2.5x2.0, GFDL-ESM2G
+    # 'gr5': 1.875x1.241, HadGEM2-ES
+    # 'gr6': 2.0x1.5, inmcm4
+    # 'gr7': 2.5x1.259, IPSL-CM5A-MR
+    # 'gr8': 2.812x2.813, MIROC-ESM
+    # 'gr9': 1.875x1.875, MPI-ESM-LR
+    # 'gr': 2.5x1.875, NorESM1-M
+    frequency: 'fx'
+
+# tau_ctotal_fx_Carvalhais2014_BE_gr5
+
+preprocessors:
+  # --------------------------------------------------
+  # preprocessors: time mean, spatial regrid, and mask
+  # --------------------------------------------------
+  preproc_meanRegrid:
+    custom_order: true
+    regrid:
+      target_grid: 0.5x0.5
+      # target_grid: 2.5x1.875
+      scheme: nearest
+    mask_landsea:
+      mask_out: sea
+    multi_model_statistics:
+      span: overlap
+      statistics: [median]
+    climate_statistics:
+      operator: mean
+      period: full
+
+diagnostics:
+  # --------------------------------------------------
+  # diagnostics: global and zonal turnover
+  # --------------------------------------------------
+  diag_land_carbon_turnover:
+    description: diagnose the turnover time of ecosystem carbon.
+    variables:
+      gpp: &variable_settings
+        # gross primary productivity
+        exp: historical
+        start_year: 1982
+        end_year: 2005
+        preprocessor: preproc_meanRegrid
+        fx_files: [sftlf, areacella]
```

Comment:
Does this work/do anything? This looks like the old syntax we had for specifying fx files.

Reply:
Apparently, it does not do anything. I wanted to use area fraction and found the above lines in the documentation. So, I thought it was part of the latest version. I have removed the line.

Reply:
You can now specify these as normal variables in the recipe and they will be included in the list of input variables/files the diagnostic script receives. Or did you want to use them with a specific preprocessor?